File: Exercicios_23.04/Python/exercicio10.py
salario_hora = float(input("Digite o salário por hora: "))

horas_trabalhadas = float(input("Digite a quantidade de horas trabalhadas: "))

salario = float(salario_hora * horas_trabalhadas)

ir = float(salario * (11/100))

inss = float(salario * (8/100))

sindicato = float(salario * (5/100))

print(f"Salário bruto: {salario}")
print(f"Quanto pagou ao INSS: {inss}")
print(f"Quanto pagou de sindicato: {sindicato}")
print(f"Salário liquido: {salario - ir - inss - sindicato}")

File: Exercicios_2304/Ruby/exercicio10.rb
puts ("Digite seu salário por hora: ")
salario_hora = gets.to_i

puts ("Digite quantas horas você trabalhou: ")
horas_trabalhadas = gets.to_i

salario = (salario_hora * horas_trabalhadas)

puts ("Seu salario é de R$ #{salario}")

ir = (salario * 11/100)

inss = (salario * 8/100)

sindicato = (salario * 5/100)

puts ("Salário bruto: #{salario}")
puts ("Quanto pagou ao INSS: #{inss}")
puts ("Quanto pagou de sindicato: #{sindicato}")
puts ("Salário liquido: #{salario - ir - inss - sindicato}")